### PR TITLE
fix(nuke): wire page-side reload listener so nuke actually reloads

### PR DESCRIPTION
**Pull request by trieloff**
## Problem

`nuke <launch-code>` (launch code: `1234`) wipes service workers, every IndexedDB database, the OPFS-backed VFS root, and a small set of `localStorage` keys — but **the page never reloads**, so to a user it looks like nothing happened.

Root cause: the agent shell runs in the kernel worker (standalone) or the offscreen document (extension), where `location.reload()` is a no-op and where worker-local `localStorage` writes never reach the page's real storage. The code already has the right fallback — a `slicc-nuke-control` BroadcastChannel and a page-side `installNukeReloadListener` — but `installNukeReloadListener()` was exported and unit-tested and then **never invoked by any bootstrap**, so the broadcast had no listener.

## Fix

- New `packages/webapp/src/ui/boot/setup-nuke-reload-listener.ts` — idempotent thin wrapper around `installNukeReloadListener()`. Module-level dispose handle means repeat calls return the same disposer and never stack listeners.
- Wire `setupNukeReloadListener()` into `packages/webapp/src/ui/main.ts` once, right after `startFreezeWatchdog()`. `main.ts` is the shared entry point for every page-owning float (standalone, electron-overlay, hosted-leader, cherry, extension side panel, extension detached popout), so one call covers them all. The `?ui-fixture` design-time surface is unaffected — it short-circuits before this stage as before.
- Out of scope on purpose: did not change the `1234` launch-code contract, did not change the set of wiped state, did not add a `--yes` flag.

## Docs

- `docs/shell-reference.md` row for `nuke` rewritten. It used to advertise a non-existent `--yes` flag; it now documents the real `nuke <launch-code>` invocation, what gets wiped, and the `slicc-nuke-control` BroadcastChannel / `setup-nuke-reload-listener.ts` wiring that drives the page reload.

## Tests

- New `packages/webapp/tests/ui/boot/setup-nuke-reload-listener.test.ts` (3 tests):
  1. After `setupNukeReloadListener()`, a `nuke-reload` broadcast triggers `location.reload()` exactly once.
  2. Repeat calls do not stack listeners or double-fire (idempotency).
  3. Repeat calls return the same dispose handle.
- Existing `packages/webapp/tests/shell/supplemental-commands/nuke-command.test.ts` continues to pass unchanged.

## Verification

```
npm run lint:ci          # passes
npm run typecheck        # passes (all tsc passes)
npm run test -w @slicc/webapp   # 6381 passed, 10 skipped, 1 file skipped
```

## Cross-runtime parity

- `webapp` standalone page: covered (single `main.ts` boot path).
- `chrome-extension` side panel + detached popout: covered (same `main.ts` boot path bundles into the extension via `packages/chrome-extension/vite.config.ts`).
- `node-server`, `swift-server`, `ios-app`: N/A — `nuke` is a browser-realm command that wipes browser-resident state (IDB / OPFS / `localStorage`); neither the Node-side server nor the native follower runtimes carry that state.

## Review-pattern checklist

- Error-path coverage: the listener already swallows individual `localStorage.removeItem` failures; the new wrapper is a single `BroadcastChannel.addEventListener` call. No new external I/O.
- UI state preservation: no DOM rewrites in scope.
- Cross-runtime parity: see the matrix above.
- CDP edge cases: N/A.
- Native/macOS permissions: N/A.
- Test coverage: regression test added for the missing wiring.

---
Pull Request opened by [Augment Code](https://www.augmentcode.com/) with guidance from the PR author

### CODE REVIEW

File: packages/webapp/src/ui/boot/setup-nuke-reload-listener.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ *
+ * Called from `main.ts` once per page-owning runtime: standalone,
+ * electron-overlay, hosted-leader, cherry, extension side panel, and
+ * the extension detached popout. Cloud / hosted floats reuse the same
+ * `main.ts` boot path so they are covered by the same call. The
+ * design-time `?ui-fixture` surface has no kernel and skips this stage
+ * along with the rest of boot.
+ */
+
+import { installNukeReloadListener } from '../../shell/supplemental-commands/nuke-command.js';
```
[the remark this answers is not in the document]
Comment (trieloff) [reply packages/webapp/src/ui/boot/setup-nuke-reload-listener.ts]: Good catch — fixed in c2e98832d.

Split the page-side wiring (`NUKE_CONTROL_CHANNEL`, `NukeReloadMsg`, `NUKE_LOCAL_STORAGE_KEYS`, `installNukeReloadListener`) into a new, zero-import `packages/webapp/src/shell/supplemental-commands/nuke-channel.ts`. `nuke-command.ts` now re-exports those symbols from the channel module (so existing consumers and the test suite keep their import paths working), and the boot helper imports straight from `nuke-channel.ts`. `main.ts` no longer has any path that reaches `defineCommand` / `just-bash`.

Verified: `npm run lint:ci`, `npm run typecheck`, and the full `@slicc/webapp` suite (6428 passed, 10 skipped) all pass.
